mentoring/README.md

```diff
@@ -6,7 +6,9 @@ The CNCF participates in and runs several mentorship programs throughout the yea
 
 ## Mission
 
-Encourage cloud native computing adoption by providing opportunities for a diverse group of new contributors to work on CNCF projects with experienced mentorship.
+* Encourage cloud native computing adoption by providing opportunities for a diverse group of new contributors to work on CNCF projects with experienced mentorship.
+* Promote growth and sustainability for projects through mentoring new and existing contributors. 
+* Provide support and advice for the projects
```

Comment:
I wonder if it's long enough we should add a short preamble before the points:
```suggestion
The Mentorship Working group
* Encourages cloud native computing adoption by providing opportunities for a diverse group of new contributors to work on CNCF projects with experienced mentorship.
* Promotes growth and sustainability of projects through mentoring new and existing contributors. 
* Provides support and advice to the projects around mentorship initiatives.
```

Reply:
I agree, the conversation originally grew from 'community responsibility' so maybe something to reflect that? Or did you have something specific in mind?

Reply:
Updated as proposed.
Yes, we could also add a preamble around 'community responsibility'